src/sass/hmda.scss

```diff
@@ -32,6 +32,11 @@ hr {
     padding: 0;
 }
 
+table td,
+table th {
+  padding: 1rem;
```

Comment:
What do you think about shrinking this even more? `.75` seems nice too. 
Also, in either case, [this](https://github.com/cfpb/hmda-platform-ui/blob/f6e3368d976fcc78a895c34ecd8b4877326a39a2/src/sass/components/IRSReport.scss#L20) can be removed.
And in the IRS file, we should add `tfoot td` [here](https://github.com/cfpb/hmda-platform-ui/blob/f6e3368d976fcc78a895c34ecd8b4877326a39a2/src/sass/components/IRSReport.scss#L18). I noticed the font size is off there.